test/stability/testSingleWorkingFF/testSingleWorkingFF.go

```diff
@@ -292,7 +289,7 @@ func setParameters(testScenario uint) (err error) {
 		}
 		// Test expects to receive 100% of packets on 0 port and 0% on 1 port (33% total)
 		lessPercent = 100
-		passedLimit = 28 // 85 * 0.33
+		passedLimit = 31 // 98 * 0.33
 		totalPackets = totalPackets / 3
```

Comment:
Can you use a named constant defined at the beginning of the file instead of magic number 98?

Reply:
It is just name inside setParameters